corehq/apps/translations/forms.py

```diff
@@ -258,3 +260,41 @@ class TransifexOrganizationForm(forms.ModelForm):
     def __init__(self, *args, **kwargs):
         super(TransifexOrganizationForm, self).__init__(*args, **kwargs)
         self.initial['api_token'] = b64_aes_decrypt(self.instance.api_token)
+
+
+class AddTransifexBlacklistForm(forms.ModelForm):
+    app_id = forms.ChoiceField(label=ugettext_lazy("Application"), choices=(), required=True)
+    action = forms.CharField(widget=forms.HiddenInput)
+    domain = forms.CharField(widget=forms.HiddenInput)
+
+    def __init__(self, domain, *args, **kwargs):
+        super(AddTransifexBlacklistForm, self).__init__(*args, **kwargs)
+        self.helper = FormHelper()
```

Comment:
Use `HQFormHelper` here and drop the `label_class` and `field_class`, unless there's a reason for this form to be unique.